R/kde.R

```diff
@@ -1,6 +1,14 @@
 kde <-
+  #'@param X an n by d matrix of coordinates used in kernel density estimation
+  #'where n is the number of points and d is the dimension
+  #'@param Grid an m by d matrix of coordinates where m is the number of points and d
+  #'is the dimension of the base grid referred to in the estimation
+  #'@param h a desired number to be used as the smoothing parameter of the Gaussian Kernel
+  #'@param kertype set by default to to use Gaussian kernel
+  #'@param weight set by default to 1, making all points in X weighted the same unless
+  #'otherwise specified
 function(X, Grid, h, kertype = "Gaussian", weight = 1, printProgress = FALSE) {
-
+  # exception handling of inputs to ensure functionality
```

Comment:
Let's remove also for a somewhat similar comment to below.  Although this comment does provide some additional information to the actual code, almost all of the functions in the package do some precondition checking.   Since each function is doing some checks, we would end up having the same comments everywhere.